### ISSUE TITLE
Automatic email sent out to contributors once the dataset has become public in the Data Portal

### ISSUE THREAD

**Issue by Wkt8** [ebi-ait/dcp-ingest-central#918]
As a data wrangler I want to inform contributors that their dataset has become public in the data portal so that they are aware that we have wrangled their paper and can bring us any updates in the future 

Tasks:
- [x] Create a template email to be sent out
- [ ] Liase with coders on when the template email should be sent out / how to get the corresponding contributors email / what cases the template email should not be sent out (e.g. updates) 
- [ ] testing
- [ ] live in production

** Acceptance Criteria / Definition of Done **

* Given a dataset has been exported from Ingest and gone live on the Data Portal for the first time 
* When the dataset goes live
* Then a template email gets sent to the corresponding contributor that their project is live on the Data Portal

**Comment by idazucchi** [comment]
@ofanobilbao to add details about ticking the catalogue option once a project is exported

**Comment by ESapenaVentura** [comment]
In the operations review, we discussed if we had a way to check if a dataset has been published in the DCP. 

Currently, we know of a way, by pinging Azul's API. An example of a Cron job that runs periodically to update the project catalogue with publication info + link to DCP portal. There are instruction on how to create other cron jobs as well https://github.com/ebi-ait/ingest-kube-deployment/tree/master/cron-jobs

If we take this path, we would need to:
- Learn how to send emails through github actions ( https://github.com/marketplace/actions/send-email ?)
- Create a script that checks which projects have been added/updated in the latest release, and sends an email to the corresponding contributor(s)
- (Optional?) that same script could add the tick to the show in catalogue?

**Comment by Wkt8** [comment]
See draft email here: https://docs.google.com/document/d/1_NrCw9Y06BtqM7ULnjw0lY1pxfg4mZogfE_7XnbljTk/edit#heading=h.gp16d9h6lxpi

**Comment by ofanobilbao** [comment]
@Wkt8 and @ESapenaVentura. Good job with the draft emails. At the bottom I've done my own version combining both of yours, as I thought both had their strengths and weaknesses. See what you think

**Comment by gabsie** [comment]
Right, I have now further edited it only to make it concise. :) Oihane, if you wished to check the suggestions:
https://docs.google.com/document/d/1_NrCw9Y06BtqM7ULnjw0lY1pxfg4mZogfE_7XnbljTk/edit?usp=sharing 

I think before we make this an automation, we should have it as a tick for the wrangler team. Thoughts?

**Comment by ofanobilbao** [comment]
@gabsie just accepted the changes. Very reasonable to shorten it. Happy with it!

**Comment by ESapenaVentura** [comment]
LGTM

**Comment by Wkt8** [comment]
Looks fine!
